internal/console/console.go

```diff
@@ -24,3 +28,28 @@ func Println(values ...any) {
 		Printf("%v\n", value)
 	}
 }
+
+func CreateProgressBar(description string) *progressbar.ProgressBar {
```

Comment:
doc comment

Reply:
addressed in https://github.com/authzed/zed/pull/365/commits/a32b659634bd53a282ab9700bea8c70a2fac159c